Canadian Championship Bidding Process initial draft. (#29)
Speedcubing-Canada/docs-speedcubing-canada: public/src/canadian-championship-bidding-policy.md

<style type="text/css">
  a[href]:after { content : "" }
</style>

Version: 1.0
Updated: May 11, 2024

# Canadian Championship Bidding Policy

Uses of the words “must”, “must not”, “should”, “should not” and “may” match [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).

Speedcubing Canada’s Board of Directors reserves the right to modify this document if a majority of not less than two-thirds (2/3) of Speedcubing Canada’s Board of Directors vote in favour of the modification.

> Canadian Championships _should_ should be conducted every 2 years, unless otherwise noted by Speedcubing Canada's Board of Directors.

Reviewer: Double should

- Every application **must** include the following noted in this list:
  - Location and Time
    - Possible dates
    - Venue 
      - The organization team **must** only have one venue being proposed in an application. Additional possible venues _may_ be added to an appendix in the application.  
    - City/Town of Venue
    - Number of days for the competition
  - Organization Team
    - Core Team Members
    - Experience of Core Team
  - Budget
    - Itemized budget that includes Cost of Venue, Proposed Registration Cost, Prizes, Miscellaneous Equipment Costs, and other expenses relevant to a WCA competition
    - Additional equipment requirements outside of the Organization Team (ex. Timers/Displays, Banners)
- Every application _may_ include the following:
  - Ease of national travel to the location
  - Availability of lodging and accommodation
  - Tourism in location
- Applications may include additional information that may be important in the decision-making process for the Canadian Championship Selection Committee

## Canadian Championship Selection Process

- Once the Application Submission Process is complete, the Canadian Championship Selection Process will begin.

> - The review process **must** consist of communications between the Canadian Championship Selection Committee and a final vote to select the host of the Canadian Championship.

Reviewer: I find 'between' confusing. I suggest 'within'

<style type="text/css">
  a[href]:after { content : "" }
</style>

Version: 1.0
Updated: May 11, 2024

# Canadian Championship Bidding Policy

Uses of the words “must”, “must not”, “should”, “should not” and “may” match [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).

Speedcubing Canada’s Board of Directors reserves the right to modify this document if a majority of not less than two-thirds (2/3) of Speedcubing Canada’s Board of Directors vote in favour of the modification.

Canadian Championships _should_ should be conducted every 2 years, unless otherwise noted by Speedcubing Canada's Board of Directors.

## Canadian Championship Selection Committee Creation

- For every Canadian Championship, one member of Speedcubing Canada's Board of Directors _should_ be selected to lead a separate Canadian Championship Selection Committee.
- The Canadian Championship Selection Committee **must** include all current members of Speedcubing Canada's Board of Directors.
- The selected leader _may_ add additional members as a part of the Canadian Championship Selection Committee.
  - Selected members **must** be Speedcubing Canada members at the time of selection.

>   - Selected members _should_ have previous experience in organizing large competitions.

Reviewer 2: Maybe strictly define what "large" means?

## Canadian Championship Timeline Development

- Submission dates for applications must be posted by the Canadian Championship Selection Committee at least 21 days before submissions are due.
  - The application submission date _may_ be moved with written notice from Speedcubing Canada’s Board of Directors.
- The Canadian Championship Selection period _should_ be a minimum of 14 days.
- The selection of a host for the Canadian Championships _should_ be concluded at least 9 months before the start of the event.

## Canadian Championship Application Submission Process

- A Canadian Championship Application will consist of a PDF document that contains information to support the Canadian Championship Selection Committee with the Canadian Championship Selection Process.

> - Every application **must** have a point person if an application is being submitted on behalf of a larger organization team.

Reviewer 2: ```suggestion 
Applications that are being submitted on behalf of a larger organization team **must** have a point person.
```

Reviewer 3: The document talks about the point person a lot, but this person only exists/is defined if the application is being submitted on behalf of a larger team. If an individual is submitting, then I assume all the requirements about the point person are intended to apply to that individual (such as "**must** be a member of Speedcubing Canada")?

Reviewer 3: ^ Was this noted? I'm still unclear.

    - City/Town of Venue
    - Number of days for the competition
  - Organization Team
    - Core Team Members
    - Experience of Core Team
  - Budget
    - Itemized budget that includes Cost of Venue, Proposed Registration Cost, Prizes, Miscellaneous Equipment Costs, and other expenses relevant to a WCA competition
    - Additional equipment requirements outside of the Organization Team (ex. Timers/Displays, Banners)
- Every application _may_ include the following:
  - Ease of national travel to the location
  - Availability of lodging and accommodation
  - Tourism in location
- Applications may include additional information that may be important in the decision-making process for the Canadian Championship Selection Committee

## Canadian Championship Selection Process

- Once the Application Submission Process is complete, the Canadian Championship Selection Process will begin.
- The review process **must** consist of communications between the Canadian Championship Selection Committee and a final vote to select the host of the Canadian Championship.

>   - The review period _may_ include meetings between the Canadian Championship Selection Committee.

Reviewer 2: ```suggestion
The review period _may_ include virtual meetings among the Canadian Championship Selection Committee.
```

>   - Meetings _may_ be proposed with each organization team before voting has begun.

Reviewer 2: ```suggestion
Virtual meetings between the Canadian Championship Selection Committee and each organization team _may_ be proposed before voting has begun.
```

    - Modifications _may_ be made to an application if it is required by the Canadian Championship Selection Committee.
- The voting process **must** be conducted using a ranked ballot system.
  - Each member of the Canadian Championship Selection Committee will receive a ballot listing all submitted applications.
  - Committee members *must* rank the bids in order of preference, assigning a unique rank to each bid.

>     - 1 will be considered the highest preference, with each descending number representing a lower preference.

Reviewer 3: Does this type of detail belong in this document? How the ranked ballot system implements these requirements should be up irrelevant here?

Author: Yeah, good point. I'll go ahead and acknowledge that we have a voting process, but the specifics aren't needed.

Reviewer 3: I just meant the fact that 1 is considered highest is an implementation detail that doesn't matter. I liked all the other points.

  - Once all committee members have submitted their ballots, the votes will be tabulated. One bid *must* receive a majority of votes to win.
    - In each round of tabulation, the bid with the fewest first-choice rankings will be eliminated, and its votes will be redistributed to the next-ranked bid on each ballot.
    - This process continues until one bid receives a majority of votes, making it the preferred choice of the committee.

>     - In the case of a time, the bid with the highest number of second-choice rankings will be declared the winner. If a tie persists, the bid with the highest number of third-choice rankings will be declared the winner. This process will continue until the tie is broken.

Reviewer 3: ```suggestion
    - In the case of a tie, the bid with the highest number of second-choice rankings will be declared the winner. If a tie persists, the bid with the highest number of third-choice rankings will be declared the winner. This process will continue until the tie is broken.
```

Reviewer 3: What if the tie is never broken?
